src/gluonts/model/forecast.py

```diff
@@ -637,10 +637,10 @@ def __repr__(self):
         )
 
     def to_quantile_forecast(
-        self, quantiles: List[Union[float, str]]
+        self, quantiles: List[str]
```

Comment:
`self.quantile` (invoked below) accepts `Union[float, str]` so this appears to be a (breaking) restriction.

Reply:
Turns out `QuantileForecast.__init__` executes `Quantile.from_str` (which only accepts str), hence this will fail with float type.

Reply:
I see! Makes sense then